File: URI/1 - INICIANTE/Python/2709 - AsMoedasDeRobbie.py
# Nao funciona

while True:
    try:
        qtd = int(input())
        vetor = []
        for z in range(0, qtd):
            vetor.append(int(input()))
        salto = int(input())

        soma = 0
        for y in range(0, qtd, salto):
            soma += vetor[y]
        cont = 0
        for z in range(1, soma+1):
            if soma % z == 0:
                cont += 1
        if cont == 2:
            print('You’re a coastal aircraft, Robbie, a large silver aircraft.')
        else:
            print('Bad boy! I’ll hit you.')
    except:
        break

File: URI/1 - INICIANTE/C/2709 - AsMoedasDeRobbie.c
#include <stdio.h>

int main(){
	int x = 0, qtd, y, salto = 0, soma, cont, vetor[30];
	while (scanf("%d", &qtd) != EOF){
		for(y = 0; y < qtd; y++){
			scanf(" %d", &vetor[y]);
		}
		scanf(" %d", &salto);
		y = qtd - 1;
		soma = 0;
		while(y >= 0){
			soma += vetor[y];
			y -= salto;
		}
		cont = 0;
		for(y = 1; y <= soma; y++){
			if(soma % y == 0){
				cont ++;
			}
		}
		if(cont == 2){
			printf("You’re a coastal aircraft, Robbie, a large silver aircraft.\n");
		}else{
			printf("Bad boy! I’ll hit you.\n");
		}
	}
	return 0 ;
}

